### PR TITLE
Fix NxFileUpload focus border on safari - RSC-1536

### PR DESCRIPTION
**Pull request by entroform**
https://issues.sonatype.org/browse/RSC-1536
https://issues.sonatype.org/browse/RSC-1468

It looks like this approach with box-shadow works and looks consistently well across different browsers.
I wonder if we should change our other implementation to use this instead of outline?

### CODE REVIEW

**Comment by EmilyBermudez88** [comment]
I've been playing around, and wondering what your thoughts would be using `border` instead of box-shadow? In order to keep the same size of the dismiss button when focused, we could just decrease the padding on the L & R.

**Comment by EmilyBermudez88** [comment]
If we keep the box-shadow solution, what are your thoughts on including NxPolicyThreatSlider in this fix as well? (as mentioned in your slack thread [here](https://sonatype.slack.com/archives/CJUF9G7T2/p1678893187164679))

**Comment by entroform** [comment]
> I've been playing around, and wondering what your thoughts would be using `border` instead of box-shadow? In order to keep the same size of the dismiss button when focused, and decrease the padding on the L & R (although I'm not sure if there was a specific reason to avoid borders in the first place)

Thanks Emily!
Yeah, I was also thinking about moving back to using border,
Although, if it turns out that this is a lot simpler than decreasing the paddings,
I don't think it hurts sticking with box shadow solution for this one

**Review by avnvbhatta — APPROVED** [review]

File: lib/src/components/NxFileUpload/NxFileUpload.scss [hunk 1]
```diff
@@ -98,20 +98,18 @@
     border-style: inherit;
```
Comment (EmilyBermudez88) [inline lib/src/components/NxFileUpload/NxFileUpload.scss:98]: If there's no border, I THINK we can get rid of `border-style` & 'border-width`

**Pull request merged**